index.html

```diff
@@ -9,11 +9,65 @@
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
-<body>
+<body>    
+    <header>
+        <nav class="topnav">
+            <img src="img/karma-logo.svg" alt="karma logo" 
+                width="30"
+                height="auto">
+            <ul class="nav-links">
+                <li><a href="#">Meet Karma</a></li>
+                <li><a href="#">How it Works</a></li>
+                <li><a href="#">Store</a></li>
+                <li><a href="#">Blog</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Login</a></li>
+            </ul>
+        </nav>
+    </header>
 
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-    <!-- All the images you need are in the 'img' folder -->
+<main>
+    <section>
+        <div class="hero">
```

Comment:
```suggestion
    <section class="hero">
```
It looks like we don't need to have both the section and div, so we can move the class to `section` and remove the `div`